tests/integration/src/test/java/org/mule/test/core/context/notification/FlowStackTestCase.java

```diff
@@ -53,8 +51,8 @@ public void flowStatic() throws Exception {
 
     assertThat(stackToAssert, not(nullValue()));
 
-    assertStackElements(stackToAssert, isFlowStackElement("flow", "/flow/processors/0"),
-                        isFlowStackElement("flowStatic", "/flowStatic/processors/0"));
+    assertStackElements(stackToAssert, isFlowStackElement("flow", "flow/processors/0"),
```

Comment:
I think so. Now it's clear that the location is a collection of parts separated by a /.